packages/sdk/test/evm/any-evm-keyless.test.ts

```diff
@@ -14,9 +17,11 @@ describe("Any EVM Keyless Deploy", async () => {
   let transactionCount: number;
 
   async function deployTieredDrop() {
-    const mockPublisher = process.env.contractPublisherAddress;
-    process.env.contractPublisherAddress =
-      "0x664244560eBa21Bf82d7150C791bE1AbcD5B4cd7";
+    const mockPublisher = getProcessEnv(CONTRACT_PUBLISHER_ADDRESS_KEY);
+    setProcessEnv(
+      CONTRACT_PUBLISHER_ADDRESS_KEY,
+      "0x664244560eBa21Bf82d7150C791bE1AbcD5B4cd7",
+    );
```

Comment:
I don't think you need to change it in tests since tests will always have `process.env`)